src/dune_rules/odoc.mli

```diff
@@ -2,6 +2,10 @@
 
 open Import
 
+module Paths : sig
+  val toplevel_index : Context.t -> Path.Build.t
```

Comment:
Is this supposed to be the location of the index file? If so could you add a comment detailing it as such?